src/ast-to-react.js

```diff
@@ -128,7 +127,7 @@ function getNodeProps(node, key, opts, renderer, parent, index) {
     case 'linkReference':
       assignDefined(
         props,
-        xtend(ref, {
+        Object.assign({}, ref, {
           href: opts.transformLinkUri ? opts.transformLinkUri(ref.href) : ref.href
         })
```

Comment:
with the broader syntax allowance, would it make sense to use:

```suggestion
        {
          ...ref,
          href: opts.transformLinkUri ? opts.transformLinkUri(ref.href) : ref.href
        }
```
?

Or even

```suggestion
        {
          ...ref,
          href: opts.transformLinkUri?.(ref.href) ?? ref.href
        }
```

Reply:
The nullish stuff is way too new I think. That’s going to trip up people.

For the spread: I’m guessing spread has the same support as `Object.assign`? 🤔 If so, that’s fine, if not, I’d rather wait.

I added `Object.assign` because `.includes` is also used in the code in other places. So this is more me streamlining the current “version” than me updating things. More changes can come with the ESM move?

Reply:
> For the spread: I’m guessing spread has the same support as Object.assign?

Not quite the same, below is the full support matrix

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#browser_compatibility

Picking Node for example `object.assign` was added in Node 4, object spread in Node 8.

Reply:
Seems that especially spread in objects is pretty late. Hmm, I’d prefer `Object.assign` for now, and modernizing this project in a next major over the summer or so